app/assets/stylesheets/common/base/authorize-api-key.scss

```diff
@@ -0,0 +1,50 @@
+.authorize-api-key {
+  max-width: 480px;
```

Comment:
It might be helpful for smaller screen sizes to add some horizontal padding, so that there is room between the content and the viewport edges.